docs/docs/getting-started/workspace.mdx

```diff
@@ -4,56 +4,107 @@ import ZoomableImage from "/src/theme/ZoomableImage.js";
+
+## Utility Buttons
+
+- **Plus (+)**: Located at the bottom of the sidebar, used to add new components to the workspace.
+- **Settings (gear icon)**: Provides access to settings for the Langflow Workspace, where users can customize their experience or manage account settings.
+
+The **Langflow Workspace** is where you assemble new flows by connecting components and run them. To get started, click on **New Project**. You can either build a flow from scratch (Blank Flow) or choose from pre-built starter examples.
+
+## Flows & Components
+
+A [flow](#flow) is a pipeline of components connected together in the Langflow Workspace.
+
+A [component](#component) is a single building block within a flow. It has inputs, outputs, and parameters that define its functionality.
+
+For example, the **OpenAI Model** is a **component** of the **Basic Prompting** flow.
 
 ## Flow
 
-A **flow** is a pipeline of components connected together in the Langflow canvas.
+A **Flow** is a sequence of components that are connected to perform a series of operations or tasks. Each flow is essentially a pipeline where data functions travel through various components, each modifying or analyzing the data according to its specific functionality.
 
-For example, the [Basic prompting](../starter-projects/basic-prompting) flow is a pipeline of four components:
+For example, the [Basic Prompting](../starter-projects/basic-prompting) flow is a pipeline of four components:
 
 <ZoomableImage
-  alt="Docusaurus themed image"
+  alt="Basic Prompting Flow"
   sources={{
     light: useBaseUrl("img/basic-prompting.png"),
     dark: useBaseUrl("img/basic-prompting.png"),
   }}
   style={{ width: "100%", maxWidth: "800px", margin: "0 auto" }}
 />
 
-In this flow, the **OpenAI LLM component** receives input (left side) and produces output (right side) - in this case, receiving input from the **Chat Input** and **Prompt** components and producing output to the **Chat Output** component.
+In this flow, the **OpenAI Model** receives input (left side) and produces output (right side) - in this case, receiving input from the **Chat Input** and **Prompt** components and passing the output to the **Chat Output** component.
 
 ## Component
 
-Components are the building blocks of flows. They consist of inputs, outputs, and parameters that define their functionality. These elements provide a convenient and straightforward way to compose LLM-based applications. Learn more about components and how they work in the LangChain [documentation](https://python.langchain.com/docs/integrations/components).
+Components are the building blocks of flows. They consist of inputs, outputs, and parameters that define their functionality. These elements provide a convenient and straightforward way to compose LLM-based applications. Learn more about components and how they work in the [Components Section](../components).
```

Comment:
This reference is raising a build error.

Reply:
Thanks for fixing this @ogabrielluiz